### PR TITLE
Quest Log example

### PR DESCRIPTION
**Pull request by wushin**

### CODE REVIEW

**Comment by mekolat** [comment]
It will require the use of clearvars to convert

**Comment by wushin** [comment]
Nope. Check the server. It's doing the bitmask there. Same ranges.

**Comment by mekolat** [comment]
Wushin yes I know this but you are using new variable names (ie VALON vs. STARTAREA) so players that already did the quest will be able to re-do it unless you use clearvars to convert and ohhhhhhhhhhhh nevermind I looked too fast

File: world/map/db/quest-log.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+//Quest Log
+//0,CommonVar,ServerVar,SHIFT,MASK
+1,TUTORIAL,STARTAREA,0,15
+2,VALON,STARTAREA,4,240
```
Comment (wushin) [inline world/map/db/quest-log.txt]: @mekolat ^ server reads these vars.
You use getq and it picks up on VALON and sets STARTAREA on the nibble 1.

File: world/map/npc/029-1/valon.txt [hunk 2]
```diff
@@ -12,7 +12,7 @@ OnInit:
 }
 function|script|ValonState
 {
-    set @valon_state, ((STARTAREA & NIBBLE_1_MASK) >> NIBBLE_1_SHIFT);
+    set @valon_state, getq("VALON");
```
Comment (mekolat) [inline world/map/npc/029-1/valon.txt]: could you make it work with ```set @val, VALON;``` or is that impossible? we can get params directly without getparam so there must be a way

Comment (wushin) [inline world/map/npc/029-1/valon.txt]: Its too ambiguous and a pain in the arse. As I want Quest Vars to start out as we aren't changing much else.

**Pull request merged**